### PR TITLE
implemented  #342

### PR DESCRIPTION
**Pull request by yinyue200**

### CODE REVIEW

**Comment by yinyue200** [comment]
@sebastienros

File: Jint/StringComparer.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Jint.Runtime.Interop
+{
+    public interface IStringComparer
```
Comment (sebastienros) [inline Jint/StringComparer.cs]: These interface and its implementation already exist in .NET, why are you reimplementing it?

Comment (yinyue200) [reply Jint/StringComparer.cs]: Ok, I will use the .NET implementation.

**Comment by yinyue200** [comment]
@sebastienros I have done

**Comment by sebastienros** [comment]
Can you target the dev branch instead, and rebase your changes, this PR is unusable in this current state, probably because of the recent changes I did on the dev branch. Thanks

**Comment by yinyue200** [comment]
@sebastienros new pr: #443

**Pull request closed without merging**